weblate_web/templates/features.html

```diff
@@ -81,7 +81,7 @@ <h3>{% trans "Branch Support" %}</h3>
                 <a href="https://docs.weblate.org/en/latest/admin/translating.html#additional">
                     <div class="icon"><img src="{% static "img/icon-context.svg" %}" /></div>
                     <h3>{% trans "Translation Context" %}</h3>
-                    <p>{% trans "Context, comments, nearby translations, images and corresponding source code for every string." %}</p>
+                    <p>{% trans "Description, nearby translations, images, related source code and dictionary entries—for every string." %}</p>
```

Comment:
It isn't strictly needed no, but since there are commas, and the last clause doesn't form a full sentence an em dash works nicely.

Reply:
What do you mean? Can you explain it in detail?

Reply:
Since there is no other good way to do "(a, a, a) for b" (or vice versa), where there would otherwise be a semicolon, there is now an em dash.
This improves the flow, and separates it as being interesting, rather than a part of a long-winded sentence.

Reply:
Got it.